packages/admin/cms-admin/src/cronJobs/CronJobsPage.tsx

```diff
@@ -1,43 +1,27 @@
-import { messages, Stack, StackPage, StackSwitch } from "@comet/admin";
-import { Domain } from "@comet/admin-icons";
-import { Typography } from "@mui/material";
-import { styled } from "@mui/material/styles";
+import { Stack, StackPage, StackSwitch } from "@comet/admin";
 import * as React from "react";
-import { FormattedMessage, useIntl } from "react-intl";
+import { useIntl } from "react-intl";
 
 import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { ContentScopeInterface, useContentScope } from "../contentScope/Provider";
 import { CronJobsGrid } from "./CronJobsGrid";
 import { JobsGrid } from "./JobsGrid";
 
-const ScopeIndicatorLabelBold = styled(Typography)`
-    && {
-        font-weight: 400;
-        padding: 0 8px 0 4px;
-        text-transform: uppercase;
-    }
-`;
+interface CronJobsPageProps {
+    renderContentScopeIndicator?: (scope: ContentScopeInterface) => React.ReactNode;
```

Comment:
Will this ever be anything different than global?

Reply:
I'm actually not sure. Maybe we don't need this. I can remove it again

Same for PublisherPage, although we'll probably remove it anyway